### ISSUE TITLE
Quest Zangarmarsh - Have you ever seen of these

### ISSUE THREAD

**Issue by Bleachlol** [Atlantiss/NetherwingBugtracker#2152]
**Description**: 
You release the murlocs on the stone and they go to different locations in the camp, some are dancing. 
**Current behaviour**:
You need to wait like 1 min or until the dancing murlocs disappears ***UNTIL you get credit for the quest.*** 

**Expected behaviour**: 
1:04, he gets credit exactly when he releases the murlocs. 

https://www.wowhead.com/quest=9816/have-you-ever-seen-one-of-these#videos:id=55248

**Server Revision**: 
2244

**Comment by zauru7** [comment]
The video you linked takes place 8 years after tbc

**Comment by Tenoutoften** [comment]
Unlike vanilla, Outlands hasn't been edited a whole lot

**Comment by Bleachlol** [comment]
Its not a big thing really, its just a small detail. Like I stated in the post, the quest works fine.

**Comment by zauru7** [comment]
> 
> 
> Unlike vanilla, Outlands hasn't been edited a whole lot

no but this is a typical change that happened during wotlk when a lot of tbc quests were made easier
I'm not saying that it's not bugged, I'm just saying that footage from cata or mop isn't really credible in these cases

**Comment by mhdemn** [comment]
I'm sure this quest got completed right after you released the murlocs. Rn in NW it takes long enough for people to assume its bugged and run away, missing the completion.

**Comment by Bleachlol** [comment]
> I'm sure this quest got completed right after you released the murlocs. Rn in NW it takes long enough for people to assume its bugged and run away, missing the completion.

No, I stood around and waited until I got credit for it. Not to be rude or anything, but how can you have such answer? If it was direct credit, then I would not report it.

**Comment by Cinx787** [comment]
Still seems to be an issue. I dropped the cage and waited like 5mins. Server rev: 2749

**Comment by devo6969** [comment]
Probably didn't need to be sticking around to get credit.

https://www.wowhead.com/quest=9816/have-you-ever-seen-one-of-these
"Large rock's coords are 26,22. If you find dancing murlocs entertaining, stick around to watch."

"My advice: Don't stick around. The fear caused by the murlocs lasts about 3 seconds and then the murlocs will dissapear, leaving all the mobs left to apoint you guility. In other words.. all mobs on you.."

**Comment by Godmatik** [comment]
Rev 4016 - still an issue
